ods_ci/tests/Tests/500__jupyterhub/autoscaling-gpus.robot

```diff
@@ -58,7 +58,8 @@ Spawn Notebook And Trigger Autoscale
     ...    of the GPU node.
     Select Notebook Image    ${NOTEBOOK_IMAGE}
     Select Container Size    Small
-    Set Number Of Required GPUs    1
+    Set NVidia GPU Accelerator
```

Comment:
you sure it shouldn't be `Set Accelerator` instead? `Set NVidia GPU Accelerator` is the keyword for JupyterSpawner